tests/s3-test.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+
+const assert = require('assert');
+const sandbox = require('sinon').createSandbox();
```

Comment:
```suggestion
const sinon = require('sinon');
```

Y directamente usas `sinon` que ya es un sandbox: https://sinonjs.org/releases/latest/sandbox/

> Since sinon@5.0.0, the sinon object is a default sandbox. Unless you have a very advanced setup or need a special configuration, you probably want to just use that one.